automates/text_reading/src/main/scala/org/clulab/aske/automates/apps/ExtractAndAssembleMentionEvents.scala

```diff
@@ -382,7 +247,6 @@ object ExtractAndAssembleMentionEvents extends App {
           val paramSetObjs = new ArrayBuffer[ujson.Value]()
```

Comment:
This "ParameterSetting" doesn't include "IntervalParameterSetting", right?

Reply:
Correct. I need to add that...